.github/workflows/build_and_test.yml

```diff
@@ -649,11 +649,11 @@ jobs:
         if [ -f ./dev/free_disk_space_container ]; then
           ./dev/free_disk_space_container
         fi
-    - name: Install Java 8
+    - name: Install Java ${{ inputs.java }}
```

Comment:
It seems easier to install Java this way, the daily test of branch-3.x all with `java: 8`